src/SFA.DAS.AANHub.Api/Controllers/ApprenticesController.cs

```diff
@@ -63,6 +66,30 @@ public async Task<ActionResult<GetApprenticeMemberResult>> GetApprentice(long ap
             return new OkObjectResult(apprenticeMemberResult);
 
         }
-    }
-}
 
+        /// <summary>
+        /// Patch an apprentice member
+        /// </summary>
+        /// <param name="userId"></param>
+        /// <param name="apprenticeid"></param>/// 
+        [HttpPatch]
+        [Route("{apprenticeid}")]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(string), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(typeof(NoContentResult), StatusCodes.Status204NoContent)]
```

Comment:
Covered for NotFound by applying [ProducesResponseType(typeof(NotFoundResult), StatusCodes.Status404NotFound)]. However no conditional check is applied for Member Deleted. I guess this will be a part of the outcome of the IsValid/Deleted discussion